### PR TITLE
Clean up licensing

### PR DESCRIPTION
**Pull request by richardfontana**
This pull request does a few things:
* Gets rid of MIT license notice in README.md and replaces with a reference to the SIL OFL 1.1 with clarification around trademarks.
* Replaces OFL.md with LICENSE (so that automated license scanning tools can more easily identify the license)
* Gets rid of the LGPL dual license since that seems unjustified
* Uses the standard text of OFL 1.1, but also makes use of the "Reserved Font Name" feature
* Fixes package.json to use a valid SPDX identifier for the license

### CODE REVIEW

**Comment by richardfontana** [comment]
I haven't looked closely at https://github.com/RedHatOfficial/RedHatFont/pull/4 but this probably addresses a superset of what's addressed there

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,94 @@
+Copyright 2019 Red Hat, Inc.,
```
Comment (starryeyez024) [inline LICENSE]: @richardfontana this should be file extension .md

File: README.md [hunk 2]
```diff
@@ -47,20 +47,6 @@ Since all the fonts available here are licensed with permission to redistribute,
...
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
\ No newline at end of file
+Licensed under the SIL Open Font License, Version 1.1, with Reserved Font Name Red Hat. 
+
+The SIL OFL does not grant any rights under trademark law and all such rights are reserved. Modified versions must be renamed to avoid use of any Red Hat trademarks, including but not limited to "Red Hat".
```
Comment (starryeyez024) [inline README.md]: This file can/should reference the license.md file:

```
See [LICENSE.md](LICENSE.md)
```

**Comment by davelab6** [comment]
> also makes use of the "Reserved Font Name" feature

I don't recommend this. Please use a `TRADEMARKS` file and a trademark notice appended to the line 1 copyright notice instead :)

**Comment by richardfontana** [comment]
> > also makes use of the "Reserved Font Name" feature
> 
> I don't recommend this. Please use a `TRADEMARKS` file and a trademark notice appended to the line 1 copyright notice instead :)

Can you explain why? I know you did so once several years ago but I've forgot your reasoning :-)

**Comment by starryeyez024** [comment]
tagging @davelab6  for the above comment ^

File: LICENSE (hunk 1 above)
Comment (richardfontana) [reply LICENSE]: As far as I can tell there's no official markdown version of SIL OFL 1.1, otherwise I would do that.

**Comment by richardfontana** [comment]
Actually @davelab6 I think regardless of your reasoning in this case I believe we should use that feature. The reason is these fonts are unusual in using the "Red Hat" name. That is Red Hat's most valuable legal asset. Since the one desirable (from a sort of superficial community perspective) license of the three that confusingly seem to currently apply happens to have that RFN feature we (Red Hat) might as well make full use of it. As an alternative you can try to convince this team to rename the fonts. They have made the informed decision to use this name.

File: LICENSE (hunk 1 above)
Comment (siwinski) [reply LICENSE]: No file extension needed... I only added one in my pull request out of habit.

**Comment by davelab6** [comment]
That's fine, but then Adobe and Google and possibly a bunch more people are going to need additional permission to use the RFN, since serving as a web font often means subsetting, and these larger companies may make other modifications. Do you have a process already set up for this, generally? I'll email you about the Google Fonts case

**Comment by davelab6** [comment]
(The reason is that Google and Adobe would need to do this anyway, for the trademark, but the RFN just invites a lot of people who are less diligent to end up falling out of compliance, and the termination clause in the OFL is old school with no pleasant gplv3 style healing, so I think it should only be used very sparingly. It's not a big problem to work with RFNs, it's just adding friction which most of the time isn't helpful because if there is a registered trademark then the naming is already subject to policy, and most people want their font name used in close derivatives like subsets.)

**Pull request merged**